shroud/ast.py

```diff
@@ -564,6 +834,117 @@ def clone(self):
     
         return new
 
+######################################################################
+
+class EnumNode(AstNode):
+    """
+        enums:
+        - decl: |
+              enum Color {
+                RED,
+                BLUE,
+                WHITE
+              }
+          options:
```

Comment:
What do these options do?

Reply:
'bar' and 'baz' are not real options. The intent was to demonstrate how an option or format string would be added.

Reply:
Thanks @ltaylor16 -- it is still not clear to me what the options do.